components/eventing-controller/pkg/backend/nats/jetstream/jetstream.go

```diff
@@ -327,7 +327,7 @@ func getStreamConfig(natsConfig env.NatsConfig) (*nats.StreamConfig, error) {
 }
 
 // syncSubscriptionFilters syncs the Kyma subscription filters with NATS subscriptions.
-func (js *JetStream) syncSubscriptionFilters(subscription *eventingv1alpha1.Subscription, log *zap.SugaredLogger) error {
+func (js *JetStream) syncSubscriptionFilters(subscription *eventingv1alpha2.Subscription, log *zap.SugaredLogger) error {
```

Comment:
(nit) shall we also improve the naming and not call them `Filters`? Instead use `Types` or `EventTypes` etc.

Reply:
yes we should! good point! I will try and get rid of all mentions of filters for JetStream.